packages/providers/src/provider.test.ts

```diff
@@ -94,57 +96,74 @@ describe('Provider', () => {
         val1: BigNumber.from(186),
         val2: BigNumber.from(0),
         val3: BigNumber.from(0),
-        pc: BigNumber.from(472),
-        is: BigNumber.from(464),
+        pc: BigNumber.from(0x01e0),
+        is: BigNumber.from(0x01d8),
       },
       {
         type: ReceiptType.Return,
         id: '0x0000000000000000000000000000000000000000000000000000000000000000',
         val: BigNumber.from(1),
-        pc: BigNumber.from(476),
-        is: BigNumber.from(464),
+        pc: BigNumber.from(0x01e4),
+        is: BigNumber.from(0x01d8),
       },
       {
         type: ReceiptType.ScriptResult,
         result: BigNumber.from(0),
-        gasUsed: BigNumber.from(0),
+        gasUsed: BigNumber.from(0x2c),
       },
     ]);
   });
 
   it('can transfer coin', async () => {
     const provider = new Provider('http://127.0.0.1:4000/graphql');
 
-    const from = '0x0101010101010101010101010101010101010101010101010101010101010101';
-    const to = genBytes32();
-    const color = '0x0000000000000000000000000000000000000000000000000000000000000000';
+    const sender = '0x0101010101010101010101010101010101010101010101010101010101010101';
+    const receiverA = genBytes32();
+    const receiverB = genBytes32();
+    const colorA = '0x0000000000000000000000000000000000000000000000000000000000000000';
+    const colorB = '0x0101010101010101010101010101010101010101010101010101010101010101';
     const amount = BigNumber.from(1);
 
-    const coins = await provider.getCoins(from, color);
-
-    expect(coins.length).toBeGreaterThan(0);
+    const coins = await provider.getCoinsToSpend(sender, [
+      { color: colorA, amount: amount.mul(2) },
+      { color: colorB, amount: amount.mul(2) },
+    ]);
 
     const response = await provider.sendTransaction({
```

Comment:
Oh, and I think you have to add some test coins to the DB so that you actually have coins returned from `getCoinstoSpend()`. Seems like that's why tests are failing.

Reply:
> [...] what do you think about having a helper method [...]
Definitely. Convenience APIs were in scope of this issue but I had to cut them.

Reply:
Sweet, that makes sense! Do you mind opening an issue so that we can keep track of it?

Reply:
#130